content/concepts/introduction/protocols/identify.md

```diff
@@ -4,4 +4,39 @@ description: "A way for nodes in libp2p to query and inform other nodes about th
 weight: 23
 ---
 
-Coming soon!
+The libp2p identify protocol is a way for nodes to query and inform other nodes
+about their metadata, such as their public key and the protocols they support.
+
+There are two variations of the protocol: `identify` and `identify/push`.
+
+The identify protocol, identified by the protocol ID `/ipfs/id/1.0.0`, is used
+to query a remote node for its metadata. To do so, a node opens a stream to the
+remote node and the remote node responds with an `Identify` message containing
+its metadata and closes the stream.
+
+The identify/push protocol, identified by the protocol ID `/ipfs/id/push/1.0.0`,
+is used to inform other nodes about changes in a node's metadata. When a
+node's metadata changes, it can "push" the new information to other nodes by opening
+a stream to each of the nodes to be updated, and sending an `Identify` message
+containing the updated metadata. Upon receiving the pushed `Identify` message, the
+remote node can update its local metadata repository with the information from the
+message, taking into account that missing fields should be ignored as the message may
+contain only partial updates.
+
+The `Identify` message contains several fields:
```

Comment:
I don't think we need to go through the fields of the protobuf. Instead, we should explain what the purpose of each of the bullet points (see above) is.